tests/lay-fxl-xhtml-icb_invalid_meta/EPUB/page_002.xhtml

```diff
@@ -0,0 +1,12 @@
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:epub="http://www.idpf.org/2007/ops" xml:lang="en">
+  <head>
+    <title>Invalid Meta — 2</title>
+    <meta name="viewport" content="width=900, height=600, something, foo" />
```

Comment:
There are a variety of other ways to produce invalid viewport metadata, including (not exhaustively):
- `"something, width=900, height=600"`
- `"width=900, something, height=600"`
- `"width=900, height=600,,"`
- `"something=width=100,width=900, height=600"`
- `"width=,width=900, height=600"`
- etc…
Should this be tested too?

Reply:
To be honest, it sounds a bit over the top to create separate tests for all these. 

I think the emphasis there is to see what the RS does if the viewport parser is indeed incorrect. I would think it is a reasonable assumption that the RS would react the same way with all the variations that you describe there (of course, one can imagine an implementation that concentrates on the particular issue in this very test, but that is not a realistic assumption). The test shows (if RS-s pass it, that is) that the particular normative statement can be implemented, that the specification is implementable by independent implementations, and, in a sense, this is the fundamental goal of the test suite (but, of course, it should also be useful for RS-s in general).

Bottom line: I think we are fine.